bloom_nofos/nofos/templates/nofos/nofo_import_overwrite.html

```diff
@@ -12,23 +12,37 @@
     <div class="usa-alert__body">
       <h4 class="usa-alert__heading">Warning</h4>
       <p class="usa-alert__text">
-        Re-importing a NOFO overwrites all existing content (but not its theme or assigned coach).
+        Re-importing a NOFO overwrites most existing content (but not its theme, assigned coach, or optionally page breaks).
       </p>
       <p class="usa-alert__text">
-        If you continue, you will lose all manual edits made since creating the NOFO.
+        If you continue, you will lose all manual edits made since creating the NOFO except for the settings preserved above.
       </p>
     </div>
   </div>
 
   {% with nofo|nofo_name as nofo_name_str %}
-    {% with "Edit  “"|add:nofo_name_str|add:"”" as back_text %}
+    {% with "Edit “"|add:nofo_name_str|add:"“" as back_text %}
       {% url 'nofos:nofo_edit' nofo.id as back_href %}
-      {% include "includes/page_heading.html" with title="Re-import “"|add:nofo_name_str|add:"”" back_text=back_text back_href=back_href only %}
+      {% include "includes/page_heading.html" with title="Re-import “"|add:nofo_name_str|add:"“" back_text=back_text back_href=back_href only %}
```

Comment:
The second quotation mark is now pointing the wrong way:

<img width="377" alt="Screenshot 2025-01-17 at 10 06 09 AM" src="https://github.com/user-attachments/assets/05cfddd2-d6af-437e-b5b3-c81fadff9288" />

We want to use this one:
- `”`

Not this one:
- `“`